AdminWebsite/AdminWebsite.UnitTests/Mappers/NotificationMappers/MapToMultiDayHearingConfirmationNotificationTests.cs

```diff
@@ -142,9 +142,64 @@ public void should_map_to_representative_confirmation_notification()
         [Test]
         public void should_map_to_joh_confirmation_notification()
```

Comment:
The test name and the `expectedNotificationType ` don't match, should the `expectedNotificationType ` be the none EJud version